HabboHotel/GameClients/GameClient.cs

```diff
@@ -84,7 +86,13 @@ public void SendWhisper(string message, int colour = 0)
 
     public void SendNotification(string message) => SendPacket(new BroadcastMessageAlertComposer(message));
 
-    public void SendPacket(IServerPacket message) => GetConnection().SendData(message.GetBytes());
+    public void SendPacket(IServerPacket message)
+    {
+        #if DEBUG

```

Comment:
Preprocessor directive is not needed, as you call Log.Debug() already, which is disabled in release (through nlog).